### ISSUE TITLE
About Crowd Pose source code

### ISSUE THREAD

**Issue by fred1912** [MVIG-SJTU/AlphaPose#285]
Hi guys,
Have you released training code for crowd pose?
Since I can't find the "loss function" part for crowd pose, i wonder if you have released that.

Regards,
Fred

**Comment by Fang-Haoshu** [comment]
@Jeff-sjtu
